### PR TITLE
feat: add husky v5 config

### PR DESCRIPTION
**Pull request by nuanyang233**
I noticed that we are using husky v5 in project, But husky config still be v4. So I add the husky v5 config for enable husky.
[husky doc](https://typicode.github.io/husky/#/)

### CODE REVIEW

File: .husky/.gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+_
```
Comment (guanbinrui) [inline .husky/.gitignore:1]: Excuse me, What does this mean?

**Review by guanbinrui — APPROVED** [review]

File: .husky/.gitignore (hunk 1 above)
Comment (nuanyang233) [reply .husky/.gitignore:1]: These are automatically generated by Husky

**Comment by zhouhanseng** [comment]
@nuanyang233 Hi, does this work now?

**Comment by nuanyang233** [comment]
> @nuanyang233 Hi, does this PR still in progress?

No，it's done

**Pull request merged**